lib/Dialect/TritonGPU/Transforms/AccelerateMatmul.cpp

```diff
@@ -144,6 +145,29 @@ class BlockedToMMA : public mlir::RewritePattern {
     }
   }
 
+  unsigned getMmaV3InstrN(tt::DotOp dotOp, unsigned currN) const {
+    if (mmaV3InstrN > 0)
+      return mmaV3InstrN;
```

Comment:
does this mean we only have one mmaV3InstrN over the whole ttgir? Although the two dots has no  dependencies?

Reply:
Good point. I get what your mean. Let me fix this.